mne_bids/mne_bids.py

```diff
@@ -35,7 +34,7 @@
                     _infer_eeg_placement_scheme, _parse_bids_filename,
                     _handle_kind)
 from .io import _parse_ext, ALLOWED_EXTENSIONS, reader
-
+from mne_bids.tsv_handler import _from_tsv, _combine, _drop, _contains_row
```

Comment:
passed my attention before. You should always use relative imports

Reply:
yeah, not sure why I did it this way...

Reply:
why are relative imports preferable?

Reply:
@sappelhoff you have taught me I should question more :)